data/global/Excel/skills.txt

```diff
@@ -100,7 +100,7 @@ Smite	97	pal	smite		150
 Might	98	pal	might		65																		73731	might	might		ln12	damagepercent	ln34																																																																																																	1	1	9		none														xx																														1	25														0	1					1	8	0	0	1	1	1		50																			16	Radius baseline	2	Radius per level	40	Damage % baseline	10	Damage % per level																	1								8																																				256	1000	0
 Prayer	99	pal	prayer		65																		73731	prayer	prayer		ln12	hitpoints	edns																																																																																																		1	7		none														xx																														1	25														0	1					1	4	16	3	1	1	1		50																			16	Radius baseline	2	Radius per level																					1								8																2	1	1	2	2	3															256	1000	0
 Resist Fire	100	pal	resist fire		65																		73731	resistfire	resistfire		ln12	fireresist	dm34	maxfireresist	skill('Resist Fire'.blvl)															passive_resistfire			maxfireresist	skill('Resist Fire'.blvl)/2																																																																												1	1	8		none														xx																														1	25														0	1					0	8	0	0	1	1	1		50																			16	Radius baseline	2	Radius per level	35	Resistance % Min	150	Resistance % Max																	1								8																																				256	1000	0
-Holy Bolt	101	pal	holy bolt		8										holybolt			holybolt																																																																												paladin_holybolt_cast											cast_undead				17						holybolt	holybolt											1	7		none												SC	SC	xx												1	1							1							paladin_holybolt_cast	cast_undead		6	25														1	1					1	4	32	1	1	1										1 + stat('extra_holybolt'.accr)	Projectile Count											1	HP Heal Min baseline	2	HP Heal Min per level	6	HP Heal Max baseline	4	HP Heal Max per level					15	Healing synergy	50	Damage synergy									1								8															mag	8	8	10	13	16	20	16	8	11	15	18	23	(skill('Blessed Hammer'.blvl)+skill('Fist of the Heavens'.blvl))*par8								384	3000	0
+Holy Bolt	101	pal	holy bolt		8										holybolt			holybolt																																																																												paladin_holybolt_cast											cast_undead				17						holybolt	holybolt											1	7		none												SC	SC	xx												1	1							1							paladin_holybolt_cast	cast_undead		6	25														1	1					1	4	32	1	1	1										1 + stat('extra_holybolt'.accr)	Projectile Count											1	HP Heal Min baseline	2	HP Heal Min per level	6	HP Heal Max baseline	4	HP Heal Max per level					15	Healing synergy	40	Damage synergy									1								8															mag	8	8	10	13	16	20	16	8	11	15	18	23	(skill('Blessed Hammer'.blvl)+skill('Fist of the Heavens'.blvl))*par8								384	3000	0

```

Comment:
This also needs skilldesc update doesn't it?

Reply:
Modification is for 'Holy Bolt'par8 which is used for both the synergy damage calc in skills.txt and the tooltip in skilldesc.txt
Tooltip is auto updated in this case since the formula didn't change, just the constant.